functest/test_test.go

```diff
@@ -61,26 +62,46 @@ func TestTestSimple(t *testing.T) {
 		log.Println("Created project dir: " + projectDir)
 
 		// appsody init
-		_, err = cmdtest.RunAppsodyCmdExec([]string{"init", stackRaw[i]}, projectDir)
 		log.Println("Running appsody init...")
+		_, err = cmdtest.RunAppsodyCmdExec([]string{"init", stackRaw[i]}, projectDir)
 		if err != nil {
 			t.Fatal(err)
 		}
 
 		// appsody test
 		runChannel := make(chan error)
 		go func() {
+			log.Println("Running appsody test...")
 			_, err = cmdtest.RunAppsodyCmdExec([]string{"test"}, projectDir)
 			runChannel <- err
 		}()
 
-		// stop and cleanup
-		func() {
-			_, err = cmdtest.RunAppsodyCmdExec([]string{"stop"}, projectDir)
-			if err != nil {
-				fmt.Printf("Ignoring error running appsody stop: %s", err)
+		waitForError := 20 // in seconds
+		stillWaiting := true
+		log.Println("Waiting to see if test will fail...")
+		for stillWaiting {
```

Comment:
Just as a suggestion, we could remove this bool, and just have `for`, then we could use `break` instead of assigning the bool to `false` to exit the "for" loop.

Reply:
I tried your suggestion, but the` break`s applied to the `select` block instead of the `for` loop.